uiowa/uiowa
# [Brand P2LB] Permissions cleanup

# How to test

```
ddev blt ds --site=brand.uiowa.edu && ddev drush @brand.local sitenow_p2lb:cleanup && ddev drush @brand.local uli /admin/people/permissions
```
- Code review
or
- Confirm that editor and webmaster have permissions to edit layout builder
